public/aboutus.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en-US">
+<head>
+  <!-- v8299 -->
+  <title>About SG Kampung &ndash; SGKampung</title>
+
+  <meta name="csrf-param" content="authenticity_token" />
+<meta name="csrf-token" content="QjMUguAV6ChhPjfBmSieN4HvizBcLU3hyEbvFl2WT7UHgDUMwGi6tIZr3vrJAzUZm3WLtw3P8FAW5cmUvpGRMg==" />
+  <meta name="description" content="What is SG Kampung?  SG Kampung is the free social network being used in thousands of Constituents across Singapore, helping residents to get the best out..." /><meta property="og:image" content="http://p4.zdassets.com/hc/settings_assets/571017/200048839/WeuCIW71z70Mkx7GbJTOFg-SG Kampung-logo-transparent-170.png" />
+<meta property="og:type" content="website" />
+<meta property="og:site_name" content="SG Kampung" />
+<meta property="og:title" content="About SG Kampung" />
+<meta property="og:description" content="What is SG Kampung?
+SG Kampung is the free social network being used in thousands of Constituents across Singapore, helping residents to get the best out of their local area.
+There are many reasons to join SG Kampung:..." />
+<meta property="og:url" content="http://help.SG Kampung/hc/en-us/articles/202364115-About-SG Kampung" />
+<link href="https://help.SG Kampung/hc/en-us/articles/202364115-About-SG Kampung" rel="canonical" />
+
+  <!-- Entypo pictograms by Daniel Bruce — www.entypo.com -->
+  <link rel="stylesheet" media="all" href="//p4.zdassets.com/hc/assets/application-39ea010cdb8a00b1e000aa73c81fcaa7.css" id="stylesheet" />
+  <link rel="stylesheet" type="text/css" href="//p4.zdassets.com/hc/themes/571017/201259329/style-85b7052657cabfcbfca48e05068f7744.css?brand_id=102805&amp;locale=en-us"></link>
+  <link rel="shortcut icon" href="//p4.zdassets.com/hc/settings_assets/571017/200048839/Dh0xsnG53xmYFvLef17EzQ-favicon.ico"></link>
+
+  <!--[if lt IE 9]>
+  <script>
+    //Enable HTML5 elements for <IE9
+    'abbr article aside audio bdi canvas data datalist details dialog \
+    figcaption figure footer header hgroup main mark meter nav output \
+    progress section summary template time video'.replace(/\w+/g,function(n){document.createElement(n)});
+  </script>
+<![endif]-->
+
+  <script src="//p4.zdassets.com/hc/assets/jquery-52fb0a6133e76d47941c21571bc3a244.js"></script>
+
+
+
+  <!-- add code here that should appear in the document head -->
+  <script type="text/javascript" src="//p4.zdassets.com/hc/themes/571017/201259329/script-85b7052657cabfcbfca48e05068f7744.js?brand_id=102805&amp;locale=en-us"></script>
+</head>
+<body class=" ">
+
+
+
+
+
+
+  <header class="header">
+  <div class="logo"><a title="Home" href="/hc/en-us">
+    <img src="../public/img/SGKampung.ico" alt="Logo">
```

Comment:
it will, many of the links will have 404, these are for me to fix through structuring of the file/icon locations. will definitely have to come to these when styling and implementing functionalities